_deployment/checklist/student_4x/index.md

```diff
@@ -28,6 +28,25 @@ Docker images are tagged with the same version as listed here.  The only project
 | Equation Scoring Service &nbsp;| [3.1.1.RELEASE](https://github.com/SmarterApp/TDS_ItemScoring/releases/tag/3.1.1.RELEASE) |
 | Configuration Service &nbsp;| [3.1.1.RELEASE](https://github.com/SmarterApp/SS_ConfigurationService/releases/tag/3.1.1.RELEASE) |
 
+# Release Notes
```

Comment:
@alexdean201 , in the past we have not added release notes to the checklist.  Would you like release notes to go within the checklist or is there somewhere else that release notes should reside?